### PR TITLE
SOC-3196 | introducing following modal

### PR DESCRIPTION
**Pull request by latata**
## Links

* https://wikia-inc.atlassian.net/browse/SOC-3196

## Description

Introducing following modal

## Reviewers

@Wikia/social-frontend

### CODE REVIEW

File: front/main/app/styles/component/discussion/_discussion-introducing-following.scss [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.introducing-following {
+
+	.header {
+		align-items: center;
+		display: flex;
+		height: 84px;
+		justify-content: center;
+
+		.follow-white {
+			fill: #fff;
```
Comment (bkoval) [inline front/main/app/styles/component/discussion/_discussion-introducing-following.scss]: If you could, define a var for it in _settings.scss and reference it here.

File: front/main/app/styles/component/discussion/_discussion-introducing-following.scss [hunk 2]
```diff
@@ -0,0 +1,50 @@
+.introducing-following {
+
+	.header {
+		align-items: center;
+		display: flex;
+		height: 84px;
+		justify-content: center;
+
+		.follow-white {
```
Comment (bkoval) [inline front/main/app/styles/component/discussion/_discussion-introducing-following.scss]: I'd rather not use color name directly in the className, I'd rather try to name it after the class purpose, not it's look, which can actually change in the future.

File: front/main/app/templates/components/modal-dialog.hbs [hunk 3]
```diff
@@ -1,4 +1,4 @@
 <div class="modal-dialog-layover" {{action 'close'}}></div>
 <div class="modal-dialog">
-	{{yield}}
+	{{yield (action 'close')}}
```
Comment (latata) [inline front/main/app/templates/components/modal-dialog.hbs]: We talked about it, works good.

File: front/main/app/templates/discussion/forum.hbs [hunk 4]
```diff
@@ -37,3 +37,11 @@
 			setEditorActive=(action 'setEditorActive')
 	}}
 {{/discussion-feed}}
+{{#modal-dialog
+		class='introducing-following-modal'
```
Comment (latata) [inline front/main/app/templates/discussion/forum.hbs]: We talked about it. It's ok

**Pull request merged**